source/SnapDump-Server/SDServer.class.st

```diff
@@ -4,7 +4,8 @@ Class {
 	#instVars : [
 		'port',
 		'server',
-		'path'
+		'path',
+		'debugMode'
```

Comment:
I see debugMode set everywhere but I don't see what it is used for ?

Reply:
It used by test to set true as value for most of the tests.
There is also one test that explicitly needs to set the value false (#testInternalServerError)